<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="L_ClientSeed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Client seed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_BrowseClientSeed">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dt_EndDate">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_FeatureIdPicker">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Feature</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_EndDate">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>End date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dt_StartDate">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="t_LicenseSeedPath">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_RemoveSelectedFromLicense">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_ProductIdPicker">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Product</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_StartDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_AddToLicense">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to license</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_GenerateLicense">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate license</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_ProductIdPicker">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_FeatureIdPicker">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_LicenseContent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
